### PR TITLE
Update ExcessiveHeaders.md

### PR DESCRIPTION
**Pull request by maybgit**
add removeServerHeader attribute description.

### CODE REVIEW

File: Extending/Health-Check/Guides/ExcessiveHeaders.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ Unless you publicly expose the Kestrel server ([not recommended by Microsoft](ht
 
 For IIS you will need to manipulate `web.config` (If you don't have `web.config` already in your project you will need to add it at the root). Ensure to remove the custom `X-Powered-By` and `Server` header as shown in the following example.
 
+Note: removeServerHeader attribute was added in IIS 10.0 and does not work in versions of Windows prior to Windows Server, version 1709 or Windows 10, version 1709.
```
Comment (eshanrnh) [inline Extending/Health-Check/Guides/ExcessiveHeaders.md:22]:
```suggestion
:::note
The `removeServerHeader` attribute is added in IIS 10.0 and does not work in versions of Windows prior to Windows Server version 1709 or Windows 10 version 1709.
:::
```

**Review by eshanrnh — APPROVED** [review]
Looks great, @maybgit 💪

Just a small suggestion with respect to formatting 😃

**Comment by eshanrnh** [comment]
For reference: https://docs.microsoft.com/en-us/iis/configuration/system.webserver/security/requestfiltering/#attributes

**Comment by maybgit** [comment]
Thanks, I will do better next time.  @eshanrnh

**Pull request merged**